scripts/systemtap/kem/kemc.c

```diff
@@ -220,7 +220,7 @@ static int kem_init(struct kemc_ctx *ctx, int dev_num,
 		    bool use_stdout, bool use_addb2)
 {
 	struct sigaction sa;
-	int              rc = 0;
+	int		 rc = 0;
```

Comment:
Rollback this. TABs are not allowed in the middle of the code.

Reply:
This code was like that before, This to align the variable name as per the coding style